multiqc/modules/nonpareil/nonpareil.py

```diff
@@ -0,0 +1,276 @@
+        """Take the parsed stats from the nonpareil report and add it to the
+        General Statistics table at the top of the report"""
+
+        headers = {
+            "L": {
+                "title": "Read Length",
+                "min": 0,
+                "suffix": " bps",
+                "scale": "GnBu",
+                "format": "{:,.3f}",
+                "hidden": True,
+            },
+            "AL": {
+                "title": "Adjusted read length",
+                "description": "Adjusted read length (if using kmers)",
+                "min": 0,
+                "scale": "GnBu",
+                "suffix": " bps",
+                "format": "{:,.3f}",
+                "hidden": True,
+            },
+            "R": {
+                "title": "Number of reads",
+                "min": 0,
+                "scale": "GnBu",
+                "format": "{:,.0f}",
+                "hidden": True,
+            },
+            "LR": {
+                "title": "Sequencing effort",
+                "description": "Effective sequencing effort used",
+                "min": 0,
+                "suffix": " Mbps",
+                "scale": "Blues",
+                "format": "{:,.2f}",
+                "hidden": True,
+            },
+            "overlap": {
+                "title": "Read overlap",
+                "description": "Minimum read overlap",
+                "min": 0,
+                "suffix": " bps",
+                "scale": "Blues",
+                "hidden": True,
+                "format": "{:,.0f}",
+            },
+            "log.sample": {
+                "title": "Log sampling",
+                "description": "Multiplier of the log-sampling (or zero if linear)",
+                "max": 1,
+                "min": 0,
+                "scale": "BuGn",
+                "hidden": True,
+                "format": "{:,.2f}",
+            },
+            "kappa": {
+                "title": "Redundancy",
+                "description": "Dataset redundancy",
+                "max": 1,
+                "min": 0,
+                "scale": "RdYlGn-rev",
+                "format": "{:,.2f}",
+            },
+            "C": {
+                "title": "Coverage",
+                "description": "Dataset coverage",
+                "max": 1,
+                "min": 0,
+                "scale": "RdYlGn",
+                "format": "{:,.2f}",
+            },
+            "consistent": {
+                "title": "Consistent Data?",
+                "description": "Is the data sufficient for accurate estimation?",
+                "modify": lambda x: "Yes" if x == 1 else "No",
+                "scale": False,
+                "hidden": True,
+            },
+            "star": {
+                "title": "Ideal coverage",
+                "description": "Coverage considered ’nearly complete’.",
+                "min": 0,
+                "suffix": "%",
+                "scale": False,
```

Comment:
Is there no scale here because the number is constant? Would be nice to add some sentence into the description about how this number is used.

Reply:
I am not even sure it makes sense to have this here, since this is actually a parameter that the user specifies when running `nonpareil`. But it reflects the objective coverage in percentage; i.e., coverage value considered near-complete.